charts/seed-bootstrap/charts/fluent-bit/values.yaml

```
@@ -16,6 +16,9 @@ ports:
   metrics: 2020
   outputPluginMetrics: 2021
 
+additionalEgressIpBlock: []
```

Nit: use plural and put it under `networkPolicy` field so that it's clear what this is used for

```suggestion
networkPolicy:
  additionalEgressIPBlocks: []
```